test/linters_test.go

```diff
@@ -99,6 +99,22 @@ func TestGciLocal(t *testing.T) {
 		ExpectHasIssue("testdata/gci/gci.go:8: File is not `gci`-ed")
 }
 
+func TestGciPanic(t *testing.T) {
```

Comment:
I don't think we need to have such tests here, it's better to move it to the cci repo. That's `//fmt` is more than enough, just put it to the `gci.go` file